rosidl_runtime_cpp/QUALITY_DECLARATION.md

```diff
@@ -99,7 +99,18 @@ The BoundedVector class is tested and the most recent test results can be found
 
 ### Coverage [4.iv]
 
-`rosidl_runtime_cpp` does not currently track test coverage.
+`rosidl_runtime_cpp` follows the recommendations for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#code-coverage), and opts to use line coverage instead of branch coverage.
+
+This includes:
+
+- tracking and reporting line coverage statistics
+- achieving and maintaining a reasonable branch line coverage (90-100%)
+- no lines are manually skipped in coverage calculations
+
+Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
+
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_rosidl_rosidl_runtime_cpp_include_rosidl_runtime_cpp/).
+A summary of how these statistics are calculated can be found in the ["ROS 2 On-boarding guide"](https://index.ros.org/doc/ros2/Contributing/ROS-2-On-boarding-Guide/#note-on-coverage-runs).
```

Comment:
@brawner consider removing the quotes around `ROS 2 On-boarding guide`.

Reply:
Done